org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -27,7 +27,8 @@
             <unit id="org.eclipse.jdt.apt.pluggable.core" version="0.0.0"/>
 			<unit id="org.eclipse.pde.junit.runtime" version="0.0.0" />
 			<unit id="org.eclipse.jdt.junit4.runtime" version="0.0.0" />
-            <repository location="https://download.eclipse.org/eclipse/updates/4.33/"/>
+			<unit id="org.eclipse.jdt.core.manipulation" version="0.0.0" />
```

Comment:
I should probably write a comment about this. Basically, when I was using releases/2024-03 from below, I noticed jdt.core.manipulation would get used from there, even though the one in I-builds was newer. When I bumped to 2024-06, it went away. Bumping to 2024-09 had the issue come back. I decided it makes perfect sense to explicitly require jdt.core.manipulation.